source/fixed_issues.rst

```diff
@@ -28,1097 +29,1490 @@ Issues Fixed in |release|
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.9                | `#1522`_ |                    |               |          | Log asynchronous responses in the API log                  |
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.9                | `#1510`_ |                    |               |          | 4.9 mvn version safeupgradeonly                            |
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.7, 4.8, 4.9      | `#1348`_ | CLOUDSTACK-9142_   | Bug           | Critical | Migrate VM changes ``xmlDesc`` in an unsafe way            |
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.9                | `#826`_  |                    |               |          | Fixed: Error given when creating VPN user in one network   |
-|                    |          |                    |               |          | if VR for an other network is stopped.                     |
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.9                | `#1498`_ | CLOUDSTACK-9352_   | Test          | Minor    | Test fails in Windows as the file separator "/" is         |
-|                    |          |                    |               |          | different from "\"                                         |
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.9                | `#1458`_ | CLOUDSTACK-9331_   | Bug           | Major    | Automation: Prepare and add the baremetal cfg to           |
-|                    |          |                    |               |          | marvin/config folder &marvin frame work changes to support |
-|                    |          |                    |               |          | baremetal advanced testcase                                |
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.9                | `#1452`_ | CLOUDSTACK-9322_   | Task          | Major    | Support for Internal LB functionality with Nuage VSP SDN   |
-|                    |          |                    |               |          | Plugin including Marvin test coverage                      |
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.9                | `#1271`_ | CLOUDSTACK-9164_   | Bug           | Major    | Consoleproxy does not preven
```

Comment:
@karuturi this should read "| 4.10                    | `#2071`_ | CLOUDSTACK-9815_   | Improvement     | Major    | Changed location of certificate service |"
the full feature isn't implemented yet